internal/cri/server/images/image_pull.go

```diff
@@ -263,8 +263,7 @@ func (c *CRIImageService) pullImageWithLocalPull(
 		}),
 	}
 
-	// Temporarily removed for v2 upgrade
-	//pullOpts = append(pullOpts, c.encryptedImagesPullOpts()...)
+	pullOpts = append(pullOpts, c.encryptedImagesPullOpts()...)
```

Comment:
While you have the hood open, would you mind removing the comment on the function implementation "temporarily removed for v2 upgrade"?